packages/ckeditor5-html-support/src/schemadefinitions.ts

```diff
@@ -279,8 +279,17 @@ export default {
 			model: 'htmlSummary',
 			view: 'summary',
 			modelSchema: {
-				allowChildren: '$text',
+				allowChildren: [
+					'htmlH1',
+					'htmlH2',
+					'htmlH3',
+					'htmlH4',
+					'htmlH5',
+					'htmlH6',
+					'$text'
+				],
 				allowIn: 'htmlDetails',
+				isLimit: true,
```

Comment:
Why is this added?

Reply:
I added it because I found that, without it, CK would remove the summary element if it contained a heading. If it works without it, great.